packages/core/src/components/text/_text.scss

```diff
@@ -6,8 +6,7 @@
 /*
 Text
 
-A component used to represent textual content.
-TODO BP add the docs you want here, I don't have the vision for this
+A component used to represent textual content with optional overflow behavior.
```

Comment:
soooo, this is not a full sentence.

How about:

"The `Text` component adds accessible overflow behavior to a line of text by conditionally adding the `title` attribute and truncating with an ellipsis when content overflows its container."